app/Http/Controllers/ListingController.php

```diff
@@ -259,4 +313,76 @@ public function rent(Request $request){
 
         return redirect()->route('listings.index');
     }
+
+    public function showAdvertiserListings(Request $request){
+        $listings= Listing::where("user_id", Auth::user()->id)
+            ->with("product");
+
+        if ($request->type != null){
+            $listings->where("type", $request->type);
+        }
+
+        if($request->sort != null && $request->has('sort')){
+            $listings = $listings->orderby('price', $request->sort);
+        }
+
+        $listings = $listings->simplePaginate(10);
+        return view("Listings.listing_list", [
+            'listings' => $listings
+        ]);
+    }
+
+    public function uploadCsvFile(Request $request){
+        $file = $request->file('csv_file');
+        $fileContents = file($file->getPathname());
+        foreach ($fileContents as $line) {
+            $getCsv = str_getcsv($line);
+            $data = [];
+
+            foreach ($getCsv as $csvItem) {
+                $data[] = explode(",", $csvItem);
+            }
```

Comment:
Waarom doe je dit nog? je maakt hier de structuur onnodig moeilijker?

structuur met deze lijn:
  0 => array:1 [▼
    0 => "supit"
  ]
  1 => array:1 [▼
    0 => "Dit beschrijving is een template voor dit product"
  ]
  2 => array:1 [▼
    0 => "set"
  ]
  3 => array:1 [▼
    0 => "1203.00"
  ]
  
  structuur zonder:
  array:4 [▼ // app\Http\Controllers\ListingController.php:346
  0 => "supit"
  1 => "Dit beschrijving is een template voor dit product"
  2 => "set"
  3 => "1203.00"
]